homeassistant/components/smhi/weather.py

```diff
@@ -139,7 +148,25 @@ def __init__(
             configuration_url="http://opendata.smhi.se/apidocs/metfcst/parameters.html",
         )
         self._attr_condition = None
-        self._attr_temperature = None
+        self._attr_native_temperature = None
+
+    @property
+    def extra_state_attributes(self) -> Mapping[str, Any] | None:
+        """Return additional attributes."""
+        if self._forecasts:
+            wind_gust = self._forecasts[0].wind_gust
+            if (wind_speed_unit := self._weather_option_wind_speed_unit) is not None:
+                wind_gust = speed_util.convert(
+                    self._forecasts[0].wind_gust,
+                    SPEED_METERS_PER_SECOND,
+                    wind_speed_unit,
+                )
+            return {
+                ATTR_SMHI_CLOUDINESS: self._forecasts[0].cloudiness,
+                ATTR_SMHI_WIND_GUST_SPEED: round(wind_gust, ROUNDING_PRECISION),
```

Comment:
I think we should consider adding support for wind gust to the base component, would you mind opening an architecture issue suggesting that @gjohansson-ST ?